pkg/image/export/main.go

```diff
@@ -65,9 +65,7 @@ fi
 set -e -x
 REGISTRY=$1
 
-docker load --input rancher-images.tar.gz
```

Comment:
Is there anyway we can not change this?  The newlines are on purpose so that the file we create has a proper new EOL.  If you need to remove the newlines from the this string then where ever this string is written to a file you need to write `\n` after it.

Reply:
@ibuildthecloud `go fmt` was throwing an error during drone build, that's why I couldn't not remove that line, but like you said I have added new line separately. Also I noticed  that the drone build has a couple of warnings like this initially
```
/tmp/go-link-559689336/000024.o: In function `_cgo_a5acef59ed3f_Cfunc_dlopen':
/tmp/go-build/cgo-gcc-prolog:76: warning: Using 'dlopen' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
/tmp/go-link-559689336/000019.o: In function `mygetgrouplist':
/workdir/go/src/os/user/getgrouplist_unix.go:15: warning: Using 'getgrouplist' in statically linked applications requires at runtime the shared libraries from the glibc version used for linking
```
But the build is still passing. Is that okay?